### PR TITLE
PEP-257 bodhi.server.services.overrides.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: bodhi/server/services/overrides.py [hunk 1]
```diff
@@ -182,11 +220,17 @@ def query_overrides(request):
...
+    Create or edit a buildroot override.
 
     This entails either creating a new buildroot override, or editing an
     existing one. To edit an existing buildroot override, the buildroot
     override's original id needs to be specified in the ``edited`` parameter.
+
+    Args:
+        request (pyramid.request): The current web request.
+    Returns:
+        basestring: A JSON response describing the new or edited override.
```
Comment (jeremycline) [inline bodhi/server/services/overrides.py]: It looks to me like this is returning a dictionary

**Review by jeremycline — CHANGES_REQUESTED** [review]
There's one docblock that looks slightly incorrect (dict rather than some JSON-serialized string). Other than that, this looks good.

:shipit:

**Comment by bowlofeggs** [comment]
@jeremycline I've corrected the docblock and rebased to resolve a merge conflict. How does it look now?

**Review by jeremycline — APPROVED** [review]
Go forth and :shipit: with my blessing

**Pull request merged**